README.md

```diff
@@ -124,80 +143,20 @@ make build && make up
 You can then connect to Airflow [locally](localhost:8080). Enable your DAG and see that it runs correctly.
 
 ### Production Setup
-When deploying to production make sure to set up the following environment
-variables:
-
-- `AWS_ACCESS_KEY_ID` -- The AWS access key ID to spin up the Spark clusters
-- `AWS_SECRET_ACCESS_KEY` -- The AWS secret access key
-- `AIRFLOW_DATABASE_URL` -- The connection URI for the Airflow database, e.g.
-  `mysql://username:password@hostname:port/database`
-- `AIRFLOW_BROKER_URL` -- The connection URI for the Airflow worker queue, e.g.
-  `redis://hostname:6379/0`
-- `AIRFLOW_BROKER_URL` -- The connection URI for the Airflow result backend, e.g.
-  `redis://hostname:6379/1`
-- `AIRFLOW_GOOGLE_CLIENT_ID` -- The Google Auth client id used for
-  authentication.
-- `AIRFLOW_GOOGLE_CLIENT_SECRET` -- The Google Auth client secret used for
-  authentication.
-- `AIRFLOW_GOOGLE_APPS_DOMAIN` -- The domain(s) to restrict Google Auth login
-  to e.g. `mozilla.com`
-- `AIRFLOW_SMTP_HOST` -- The SMTP server to use to send emails e.g.
-  `email-smtp.us-west-2.amazonaws.com`
-- `AIRFLOW_SMTP_USER` -- The SMTP user name
-- `AIRFLOW_SMTP_PASSWORD` -- The SMTP password
-- `AIRFLOW_SMTP_FROM` -- The email address to send emails from e.g.
-  `telemetry-alerts@workflow.telemetry.mozilla.org`
-- `URL` -- The base URL of the website e.g.
-  `https://workflow.telemetry.mozilla.org`
-- `DEPLOY_ENVIRONMENT` -- The environment currently running, e.g.
-  `stage` or `prod`
-
-
-Also, please set
-
-- `AIRFLOW_SECRET_KEY` -- A secret key for Airflow's Flask based webserver
-- `AIRFLOW__CORE__FERNET_KEY` -- A secret key to saving connection passwords in the DB
+This repository was structured to be deployed using the [offical Airflow Helm Chart.](https://airflow.apache.org/docs/helm-chart/stable/index.html).
+See the [Production Guide](https://airflow.apache.org/docs/helm-chart/stable/production-guide.html) for best practices.
 
 ### Debugging
 
 Some useful docker tricks for development and debugging:
 
 ```bash
-# Stop all docker containers:
-docker stop $(docker ps -aq)
+make clean
 
 # Remove any leftover docker volumes:
 docker volume rm $(docker volume ls -qf dangling=true)
 
-# Purge docker volumes (helps with mysql container failing to start)
+# Purge docker volumes (helps with postgres container failing to start)
 # Careful as this will purge all local volumes not used by at least one container.
 docker volume prune
 ```
-
-### Triggering a task to re-run within the Airflow UI
```

Comment:
why are we rm'ing this?

Reply:
Tree view is no longer available. I would generally refer to the Airflow documentation directly for this kind of knowledge since they keep it up to date.